Ginger/GingerCore/ALM/RQM/ExportToRQM.cs

```diff
@@ -186,9 +196,18 @@ public bool ExportExecutionDetailsToRQM(BusinessFlow businessFlow, ref string re
 
                         if (!exeResultList.Any())
                         {
-                            Reporter.ToLog(eLogLevel.DEBUG, $"Skippinng ALM Results Publish of '{businessFlow.Name}' Flow and testplan '{bfExportedID}' as no valid Execution found for it");
-                            result = $"Skippinng ALM Results Publish of '{businessFlow.Name}' Flow and 'testplan {bfExportedID}' as no valid Execution found for it ";
-                            return false;
+                            if(isFlowskipped)
+                            {
+                                Reporter.ToLog(eLogLevel.INFO, $"Skippinng ALM Results Publish of '{businessFlow.Name}' Flow and testplan '{bfExportedID}' as skippedUpdate configured as {ALMCore.DefaultAlmConfig.IsSkippedUpdate}");
+                                result = $"Skippinng ALM Results Publish of '{businessFlow.Name}' Flow and 'testplan {bfExportedID}' as skippedUpdate configured as {ALMCore.DefaultAlmConfig.IsSkippedUpdate}";
```

Comment:
no need to add it to results as it will be considered as error in Publish operations

Reply:
Updated